library/build.gradle

```diff
@@ -69,6 +69,16 @@ android {
         noStdlib = true
     }
 }
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        freeCompilerArgs += [
+                '-progressive',
+                '-Xexplicit-api=strict',
```

Comment:
Any way do disable explicit api mode for unit tests?

Reply:
There's supposed to be, but when I tried adding these arguments only for the non-test compilation tasks, the IDE still applied explicit mode in tests and produced inspection errors. I can't find the discussion I read about it then, but there's an open ticket for it [here](https://youtrack.jetbrains.com/issue/KT-42718). For now, marking all tests `internal` was the quickest solution.